jobs.groovy

```diff
@@ -200,13 +200,13 @@ pipelineJob('rootfs-build-trigger') {
       lightweight(true)
       scm {
         git {
-          branch(KCI_CORE_BRANCH)
+          branch(KCI_JENKINS_BRANCH)
           remote {
-            url(KCI_CORE_URL)
+            url(KCI_JENKINS_URL)
```

Comment:
`KCI_JENKINS_URL` and `KCI_JENKINS_BRANCH` are not defined.  They need to be added to the list of job parameters, and any default values need to be defined at the top of this file.